pkg/collector/runner/runner_test.go

```diff
@@ -176,3 +176,9 @@ func TestStopCheck(t *testing.T) {
 	err = r.StopCheck(c2.ID())
 	assert.Equal(t, "timeout during stop operation on check id TestCheck:2", err.Error())
 }
+
+func TestGetCheckStats(t *testing.T) {
+	m := GetCheckStats()
+	require.True(t, m["TestCheck"] != nil, "should be a TestCheck check")
+	require.True(t, m["TestCheck"]["TestCheck:1"] != nil, "should be a TestCheck:1 check")
```

Comment:
This test feels like it's racing with the others, updating the global `checkStats`.  But, it seems to pass reliably for me.  Advice is welcome :)